src/includes/sourcemaps/generate/javascript.mdx

```diff
@@ -0,0 +1,166 @@
+
+module.exports = {
+  // other webpack configuration
+  devtool: 'source-map',
+  plugins: [
+    new SentryWebpackPlugin({
+      // sentry-cli configuration
+      authToken: process.env.SENTRY_AUTH_TOKEN,
+      org: "___ORG_SLUG___",
+      project: "___PROJECT_SLUG___",
+      release: process.env.SENTRY_RELEASE,
+
+      // webpack-specific configuration
+      include: ".",
+      ignore: ["node_modules", "webpack.config.js"],
+    }),
+  ],
+};
+```
+
+</PlatformSection>
+
+<PlatformSection supported={["javascript.nextjs"]}>
+
+You don't need to install the Webpack plugin if you are using Sentry's Next.JS SDK because the SDK automatically handles and uploads your source maps without requiring any configuration. 
+
+If you want to extend the default Next.JS SDK configuration, you can refer [here](/platforms/javascript/guides/nextjs/manual-setup/).
+
+</PlatformSection>
+
+<PlatformSection supported={["javascript.vue"]}>
+
+In Vue 2.x, you should use `vue.config.js` instead of `webpack.config.js`, and use `include: "./dist"` instead of `include: "."`
+
+</PlatformSection>
+
+<Note>
+
+Set up the Webpack plugin as the last running plugin; otherwise, the resulting source maps the plugin receives may not be the final one.
+
+</Note>
+
+### Advanced Usage
+
+If you prefer to upload source maps manually, configure Webpack to output source maps:
+
+<PlatformContent includePath="configuration/sourcemaps/tools/webpack/advanced-usage" />
+
+If you use [SourceMapDevToolPlugin](https://webpack.js.org/plugins/source-map-dev-tool-plugin) for more fine-grained control of source map generation, turn off `noSources` so Sentry can display proper source code context in event stack traces.
+
+Additionally, the Webpack plugin will automatically set `window.SENTRY_RELEASE`, so your `Sentry.init` call will not need to be updated.
+
+## Rollup
+
+You can configure Rollup to generate source maps:
+
+```javascript {filename:rollup.config.js}
+export default {
+  entry: "./src/app.js",
+  output: {
+    file: "bundle.js",
+    format: "cjs",
+    sourceMap: true,
+  },
+};
+```
+
+<PlatformSection notSupported={["javascript.nextjs"]}>
+
+## SystemJS
+
+SystemJS is the default module loader for Angular 2 projects. The [SystemJS build tool](https://github.com/systemjs/builder) can be used to bundle, transpile, and minify source code for use in production environments, and can be configured to output source maps.
+
+```javascript
+builder.bundle("src/app.js", "dist/app.min.js", {
+  minify: true,
+  sourceMaps: true,
+  sourceMapContents: true,
+});
+```
+
+<Note>
+
+This example configuration will inline your original, un-transformed source code into the generated source map file. Sentry requires both source map(s) **and** your original source files to perform reverse transformations. If you choose NOT to inline your source files, you must make those source files available to Sentry _in addition_ to your source maps (see below).
+
+</Note>
+
+</PlatformSection>
```

Comment:
Got rid of the platform section (since nextjs shouldn't have this page at all) and took out the blurb about SystemJS. (See notes about both of these in previous comments.)

NB: Also see note above re: backticks.

```suggestion
## SystemJS

[SystemJSl](https://github.com/systemjs/builder) can be configured to output source maps, which you can then [upload using `sentry-cli`](/product/cli/releases/#sentry-cli-sourcemaps):

```javascript
builder.bundle("src/app.js", "dist/app.min.js", {
  minify: true,
  sourceMaps: true,
  sourceMapContents: true,
});
// TODO: uncomment the backticks below
// ```

<Note>

This example configuration will inline your original, un-transformed source code into the generated source map file. Sentry requires both source map(s) **and** your original source files to perform reverse transformations. If you choose NOT to inline your source files, you must make those source files available to Sentry _in addition_ to your source maps (see below).

</Note>
```